.github/workflows/e2e.yaml

```diff
@@ -1,5 +1,14 @@
+# The Tests / E2E workflow is used to run end-to-end tests on pull requests originating
```

Comment:
I think it makes more sense to add the change to run hermes here instead of duplicating all of the workflow